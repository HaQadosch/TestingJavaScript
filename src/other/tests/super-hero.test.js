import { getFlyingSuperHeros } from '../super-hero'

test('should return superheros who can fly.', () => {
  const flyingHeros = getFlyingSuperHeros()
  console.log({ flyingHeros })

  expect(flyingHeros).toEqual([
    { 'name': 'Dynaguy', 'powers': ['disintegration ray', 'fly'] },
    { 'name': 'Apogee', 'powers': ['gravity control', 'fly'] }
  ])
})
