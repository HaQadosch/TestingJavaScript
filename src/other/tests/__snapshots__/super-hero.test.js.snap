// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return superheros who can fly. 1`] = `
Array [
  Object {
    "name": "Dynaguy",
    "powers": Array [
      "disintegration ray",
      "fly",
    ],
  },
  Object {
    "name": "Apogee",
    "powers": Array [
      "gravity control",
      "fly",
    ],
  },
]
`;
